packages/@webex/plugin-meetings/src/meeting/in-meeting-actions.ts

```diff
@@ -39,6 +39,10 @@ interface IInMeetingActions {
   waitingForOthersToJoin?: boolean;
   canEnableReactions?: boolean;
   canSendReactions?: boolean;
+  canManageBreakout?: boolean;
+  canAdmitLobbyToBreakout?: boolean;
+  enabledBreakoutPreassignments?: boolean;
```

Comment:
name of this should be: `isBreakoutPreassignmentsEnabled`

Reply:
Got, will change.